openmdao/core/test/test_residual_sign.py

```diff
@@ -9,10 +9,10 @@
 from openmdao.test.util import assert_rel_error
 
 
-class SimpleImplicit(Component):
+class SimpleImplicitSL(Component):
```

Comment:
it's not clear to me what this name change means (SL = Solve Linear) and why it was deemed an improvement.. perhaps a doc string on the class would be helpful.

Also, since this class is now used in multiple tests, maybe it should be moved to one of the test component modules.. it seems odd to import a test component from another `test_*` module.